.scalafmt.conf

```diff
@@ -1,6 +1,49 @@
-version = "2.6.3"
-align.preset = more
+version = 3.7.15
+runner.dialect = scala3
 maxColumn = 110
-spaces.inImportCurlyBraces = true
-rewrite.rules = [SortImports, RedundantParens, SortModifiers]
-rewrite.redundantBraces.stringInterpolation = true
+
+newlines.source = keep
+
+trailingCommas = "multiple"
+
+rewrite {
+  rules = [Imports]
+  imports.sort = ascii
+}
+
+rewrite.scala3 {
+  convertToNewSyntax = yes
+  removeOptionalBraces = yes
+}
+
+align {
+  allowOverflow = true
+  preset = more
+  openParenCallSite = false
+  stripMargin = true
+}
+
+continuationIndent {
+  callSite = 2
+  defnSite = 4
+}
+
+docstrings {
+  style = Asterisk
+  oneline = keep
+  wrap = no
+}
+
+spaces {
+  beforeContextBoundColon = Never
+  inImportCurlyBraces = true
+}
```

Comment:
why so many specific rules? We should probably normalize our scalafmt configs

Reply:
I copy this from my other repo. in which, I tried to hard to make the code look the way I like. But agree, that we should have a consistent style for all our code. Will pick the config from lila.

Reply:
Fixed: [0ac713f](https://github.com/lichess-org/lila-fishnet/pull/247/commits/0ac713fc69c0f7ec95a7bc962103c2e86d18fa6d)